### PR TITLE
Code generated using native OpenAPI specification

### PR DESCRIPTION
**Pull request by saimedhi**
### Description
Code generated using native OpenAPI specification

### Issues Resolved
Related to https://github.com/opensearch-project/opensearch-py/issues/636
Related to https://github.com/opensearch-project/opensearch-py/issues/474

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by saimedhi** [comment]
This is generated code. 

@dblock, please review. Kindly approve if everything looks correct, but please do not merge. I'll give it one last check to make sure we haven't missed any minor details. Thank you!

**Review by dblock — COMMENTED** [review]
This looks good, and it adds the notifications plugin support. Should be in CHANGELOG at least.

**Review by dblock — DISMISSED** [review]

**Comment by saimedhi** [comment]
> This looks good, and it adds the notifications plugin support. Should be in CHANGELOG at least.

In the follow-up PR, I'll update the generator to include any newly added namespaces in the changelog.

File: utils/templates/base
Comment (saimedhi) [inline utils/templates/base:26]: Made small modification to generator

**Review by dblock — DISMISSED** [review]

**Comment by saimedhi** [comment]
@dblock, apologies for the inconvenience. I need your approval one last time before merging. Thank you.

**Review by dblock — APPROVED** [review]

**Pull request merged**
